### PR TITLE
when constructing a stream, it might not have a FDEV -- allow it to print

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

File: sources/FILEIO [hunk 1]
```diff
@@ -668,22 +666,29 @@ Copyright (c) 1981-1993, 1999, 2020-2022 by Venue & Xerox Corporation.
...
+                                                                                            )
+                                                                                 of STRM)))
+                                                  ((TYPE? FDEV (FETCH DEVICE OF STRM))
+                                                             (* ; "Name the device")
+                                                   (CONCAT TYPE " "
+                                                          [CL:STRING-CAPITALIZE
+                                                           (STRING (fetch (FDEV DEVICENAME)
+                                                                      of (fetch DEVICE of STRM]
+                                                          " Stream"))
+                                                  (T (CONCAT TYPE " unknown stream"])
```
Comment (nbriggs) [inline sources/FILEIO:691]: perhaps " unknown Stream" (capital S).  But is "unknown" the right word, or should it just be " Stream"?

**Review by nbriggs — CHANGES_REQUESTED** [review]
Other than resolving the comment,  it looks good.

File: sources/FILEIO (hunk 1 above)
Comment (masinter) [reply sources/FILEIO]: just Stream

**Review by nbriggs — APPROVED** [review]

**Review by nbriggs — APPROVED** [review]
LGTM.

**Pull request merged**